tests/io/test_file.py

```diff
@@ -213,6 +217,24 @@ def test_set_index(self):
         assert pfs.index.name is None
 
 
+@pytest.fixture
+def example_locations():
+    """Locations to load into the database."""
+    p1 = Point(8.5067847, 47.4)
+    p2 = Point(8.5067847, 47.5)
+    p3 = Point(8.5067847, 47.6)
+
+    list_dict = [
+        {"user_id": 0, "center": p1},
+        {"user_id": 0, "center": p2},
+        {"user_id": 1, "center": p3},
+    ]
+    locs = gpd.GeoDataFrame(data=list_dict, geometry="center", crs="EPSG:4326")
+    locs.index.name = "id"
+    assert locs.as_locations
```

Comment:
We once realized to only write `locs.as_locations` (delete the assert), see #215 and #255